.circleci/config.yml

```diff
@@ -0,0 +1,66 @@
+version: 2.1 # use CircleCI 2.1
+orbs:
+   node: circleci/node@5.0.0 # Needed for javascript runtime
+   heroku: circleci/heroku@1.0.1 # Use the Heroku orb in your config
+jobs: # a collection of steps
+  build-and-test: # runs not using Workflows must have a `build` job as entry point
+    docker: # run the steps with Docker
+      - image: cimg/ruby:3.2.2 # circleci/ruby:2.7.2 # if you're using ruby version >= 3.0, use cimg/ruby:3.x.y 
+        environment: # environment variables for primary container
+          BUNDLE_JOBS: 3
+          BUNDLE_RETRY: 3
+          BUNDLE_PATH: vendor/bundle
+          PGHOST: 127.0.0.1
+          PGUSER: postgres
+          RAILS_ENV: test
+      - image: circleci/postgres:11 # database image
```

Comment:
Nice job trying postgress: 14.  we will have to look into why that part wasn't working and where.